scripts/config-excursion.json

```diff
@@ -0,0 +1,28 @@
+{
+  "include": ["./src/themes/excursion/color/*.json"],
```

Comment:
```suggestion
```



Using `include` and `source` is not necessary. 

`include` is optional by definition of the API. 

> An array of file path globs to design token files that contain default styles. Style Dictionary uses this as a base collection of design tokens. The tokens found using the "source" attribute will overwrite tokens found using include.

While `source` is required

> An array of file path globs to design token files. Style Dictionary will do a deep merge of all of the token files, allowing you to organize your files however you want.

Reply:
Where this really comes in handy is if we were to want to output all the tokens into a single file where we wanted Auro classic and then a theme to override the values as written. If you include all the files into one glob, then there are duplication warnings.